android/build.gradle

```diff
@@ -10,8 +10,12 @@ android {
     }
 }
 
+def safeExtGet (prop, fallback) {
+    rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
+}
+
 dependencies {
     implementation 'com.facebook.react:react-native:[0.40,)'
     implementation 'com.urbanairship.android:urbanairship-fcm:9.5.6'
-    implementation 'com.google.firebase:firebase-core:16.0.5'
+    implementation "com.google.firebase:firebase-core:${safeExtGet('firebaseCoreVersion', '16.0.5')}"
```

Comment:
16.0.6 is out, should that be the default?

Reply:
Yeah probably. In that case shall we leave the default as `+`?

Reply:
no, use of `+` is usually frowned upon. It makes it difficult knowing exact version of dependencies in build